lesson 10/index.html

```diff
@@ -0,0 +1,320 @@
+            <stop stop-color="#32A071" />
+            <stop offset="0.0685" stop-color="#2DA771" />
+            <stop offset="0.4762" stop-color="#15CF74" />
+            <stop offset="0.8009" stop-color="#06E775" />
+            <stop offset="1" stop-color="#00F076" />
+          </linearGradient>
+        </defs>
+      </svg>
+      <svg class="triphouse__img-as">
+        <use href="./images/sprite.svg#triphouse-as" />
+      </svg>
+    </div>
+  </div>
+</div>
+<div class="offer">
+  <div class="offer__head">
+    <h2 class="offer__title">What do we offer</h2>
+    <div class="offer__line"></div>
+  </div>
+  <div class="offer__block">
+    <div class="col-lg-3 col-xs-3 col-md-5">
+      <div class="offer__subblock">
+        <svg class="offer__vector">
+          <use href="./images/sprite.svg#offer-support" />
+        </svg>
+        <h3 class="offer__text">Support 24/7</h3>
+      </div>
+    </div>
+    <div class="col-lg-3 col-xs-3 col-md-5">
+      <div class="offer__subblock">
+        <svg class="offer__vector vector-communicate">
+          <use href="./images/sprite.svg#offer-communicate" />
+        </svg>
+        <h3 class="offer__text">Communicate directly</h3>
+      </div>
+    </div>
+    <div class="col-lg-3 col-xs-3 col-md-5">
+      <div class="offer__subblock">
+        <svg class="offer__vector">
+          <use href="./images/sprite.svg#offer-book" />
+        </svg>
+        <h3 class="offer__text">Book online</h3>
+      </div>
+    </div>
+    <div class="col-lg-3 col-xs-3 col-md-5">
+      <div class="offer__subblock  ">
+        <svg class="offer__vector">
+          <use href="./images/sprite.svg#offer-reviews" />
+        </svg>
+        <h3 class="offer__text">Real guest reviews</h3>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="homes__container">
+
+  <h2 class="homes__title">Homes guests loves</h2>
+  <div class="homes__block lg-4-col">
+    <div class="homes__pointer">
+      <img src="images/polygon.png" alt="pol">
+    </div>
+    <div class="homes__subblock">
+      <a href="#">
+        <img src="./images/hotel_leopold.jpg" alt="Hotel Leopold" class="homes_img">
+      </a>
+      <a href="#" class="homes_signature">Hotel Leopold</a>
+      <h5 class="homes_text">Saint Petersburg, Russia</h5>
+    </div>
+    <div class="homes__subblock">
+      <a href="#">
+        <img src="./images/apartment_sunshine.jpg" alt="Apartment Sunshine" class="homes_img">
+      </a>
+      <a href="#" class="homes_signature">Apartment Sunshine</a>
+      <h5 class="homes_text">Santa Cruz de Tenerife, Spain</h5>
+    </div>
+    <div class="homes__subblock">
+      <a href="#">
+        <img src="./images/villa_kunerad.jpg" alt="Villa Kunerad" class="homes_img">
+      </a>
+      <a href="#" class="homes_signature">Villa Kunerad</a>
+      <h5 class="homes_text">Vysokie Tatry, Slowakia</h5>
+    </div>
+    <div class="homes__subblock">
+      <a href="#">
```

Comment:
I would recommend remove tag a for wrapper for tag img and replace to cursor:pointer in css